### PR TITLE
created advanced variant for skillset component

### PR DESCRIPTION
**Pull request by Chiraglr**
added star rating imagery below skillset.

### CODE REVIEW

**Comment by ManojSatishkumar** [comment]
@Chiraglr , Can you add a screenshot in the description ?

**Comment by ManojSatishkumar** [comment]
What is the size of render.js file ?

File: src/modules/resume/skill-set/variants/advanced/contributors.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const contributors = [
+    {
+      name: "Chirag R",
+      email: "chiraglr74@gmail.com",
+      url: "https://github.com/Chiraglr"
+    }
+  ]
+  
+  export default contributors;
```
Comment (venkateshpachigulla) [inline src/modules/resume/skill-set/variants/advanced/contributors.js:9]: Add new line at the end of each file.

File: src/modules/resume/skill-set/variants/advanced/properties.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import propertyTypes from 'msk-property-types'
```
Comment (venkateshpachigulla) [inline src/modules/resume/skill-set/variants/advanced/properties.js:1]: Add semi-colons, apply formatting for all files.

File: src/modules/resume/skill-set/variants/advanced/render.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  ratingIcon: {
+      height: 13
+  },
+  ratingContainer: {
+      paddingLeft: 5
+  }
+})
+
+const getStars = (rating) => {
+    let Stars = [];
```
Comment (venkateshpachigulla) [inline src/modules/resume/skill-set/variants/advanced/render.js:32]: rename to `stars` and format file.

**Comment by Chiraglr** [comment]
Size of render.js file is 670kb because I used base64 representation for the star rating. Here is the screenshot.
image

File: src/modules/resume/skill-set/variants/advanced/properties.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  skills: {
+    name: 'Skills',
+    type: propertyTypes.ARRAY,
+    value: [
+      {
+        name: 'Skill',
+        type: propertyTypes.OBJECT,
+        value: {
+            name: 'Skill 1',
+            rating: 1.5
```
Comment (ManojSatishkumar) [inline src/modules/resume/skill-set/variants/advanced/properties.js:26]: Each one of the value is itself a property.
We need to use only 'value' inside the render.js

`skills: {
    name: 'Skills',
    type: propertyTypes.ARRAY,
    value: [
      {
        name: 'Skill',
        type: propertyTypes.OBJECT,
        required: true,
        name: {
          name: 'Skill',
          type: propertyTypes.STRING,
          value: 'Skill 1',
          required: true
        },
        rating: {
          name: 'rating',
          type: propertyTypes.NUMBER,
          value: 1.5,
          required: true
        }
      },`

Comment (ManojSatishkumar) [reply src/modules/resume/skill-set/variants/advanced/properties.js:26]: Refer to the sample - https://github.com/ManojSatishkumar/msk-components/commit/76911f0d3781797bc03635d5c1750ba0fc01d4e2

Comment (Chiraglr) [reply src/modules/resume/skill-set/variants/advanced/properties.js]: Have pushed the changes.

File: src/modules/resume/skill-set/variants/advanced/render.js (hunk 3 above)
Comment (Chiraglr) [reply src/modules/resume/skill-set/variants/advanced/render.js]: updated with all requested changes.

File: src/modules/resume/skill-set/variants/advanced/properties.js (hunk 4 above)
Comment (Chiraglr) [reply src/modules/resume/skill-set/variants/advanced/properties.js]: @ManojSatishkumar Please let me know if changes are still required.

Comment (ManojSatishkumar) [reply src/modules/resume/skill-set/variants/advanced/properties.js]: Good.

**Comment by ManojSatishkumar** [comment]
@Chiraglr :  Once you address all comments, I will merge

File: src/modules/resume/skill-set/variants/advanced/properties.js (hunk 2 above)
Comment (Chiraglr) [reply src/modules/resume/skill-set/variants/advanced/properties.js:1]: Have pushed the changes.

File: src/modules/resume/skill-set/variants/advanced/contributors.js (hunk 1 above)
Comment (Chiraglr) [reply src/modules/resume/skill-set/variants/advanced/contributors.js]: Added new lines to the files.

**Comment by RisingGeek** [comment]
We can use Material UI's rating. What are your thoughts @Chiraglr @ManojSatishkumar ? This will not require base64.
Refer - [https://material-ui.com/components/rating/](url)

**Pull request merged**